packages/react-accordion/src/components/Accordion/useAccordion.ts

```diff
@@ -32,13 +40,15 @@ export const useAccordion = (
   });
 
   return {
-    ...rest,
-    ref,
     multiple,
     collapsible,
     navigable,
     openItems,
     requestToggle,
+    root: getNativeElementProps('div', {
+      ...rest,
```

Comment:
```suggestion
      ...props,
```

Can we spread `props` there?
Let's also move destruction of `props` into function body as in other files.